src/app/clusters/identify-server/identify-server.cpp

```diff
@@ -179,9 +179,7 @@ void MatterIdentifyClusterServerAttributeChangedCallback(const app::ConcreteAttr
                 /* finish identify process */
                 if (EMBER_ZCL_IDENTIFY_EFFECT_IDENTIFIER_FINISH_EFFECT == identify->mCurrentEffectIdentifier && identifyTime > 0)
                 {
-                    (void) chip::DeviceLayer::SystemLayer().StartTimer(System::Clock::Seconds16(1), onIdentifyClusterTick,
-                                                                       identify);
-                    return;
+                    Clusters::Identify::Attributes::IdentifyTime::Set(endpoint, 1);
```

Comment:
This might be correct but it seems it can be breaking other platforms behaviour as they would expect the callback. 

I'll take a deeper look tomorrow. 

I'll request change just so I can get a chance to fully review it at my desk tomorrow without it being merge